src/cdk/overlay/dispatchers/overlay-keyboard-dispatcher.spec.ts

```diff
@@ -192,11 +192,6 @@ describe('OverlayKeyboardDispatcher', () => {
     expect(appRef.tick).toHaveBeenCalledTimes(0);
     dispatchKeyboardEvent(document.body, 'keydown', ESCAPE);
     expect(appRef.tick).toHaveBeenCalledTimes(0);
-
```

Comment:
Should we make the same change as #28806 to capture the same behavior as before?

Reply:
My understanding is that in zoneless `NgZone.run` will not cause change detection to be run here. Currently it does, but that's just sort of incidental, nothing we're doing requires it to be run. It _is_ possible that users may be listening to the detach events and doing something that requires change detecting, but in that case their code should trigger it.

cc @atscott

Reply:
I don't feel strongly about it, just making sure we don't lose coverage.

Reply:
Yea, it's a bit hard to tell what the intent is exactly but it does seem to be more along the lines of "we don't want to cause excessive change detection" rather than "we definitely want to be triggering change detection"